src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -5229,7 +5229,7 @@ internal InterceptableLocation GetInterceptableLocationInternal(SyntaxNode nameS
             var lineNumberOneIndexed = lineSpan.Line + 1;
             var characterNumberOneIndexed = lineSpan.Character + 1;
 
-            return new InterceptableLocation1(checksum, path, nameSyntax.Position, lineNumberOneIndexed, characterNumberOneIndexed);
+            return new InterceptableLocation1(checksum, path, Compilation.Options.SourceReferenceResolver, nameSyntax.Position, lineNumberOneIndexed, characterNumberOneIndexed);
```

Comment:
A bit annoying that this is a nullable reference type when we could have used `SourceFileResolver.Default`. Not sure there is a ton of value in changing it now.